contracts/deposit-contract/sendDepositTx/BUILD.bazel

```diff
@@ -0,0 +1,61 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+load("@io_bazel_rules_docker//container:container.bzl", "container_push")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["sendDeposits.go"],
+    importpath = "github.com/prysmaticlabs/prysm/contracts/deposit-contract/sendDepositTx",
+    visibility = ["//visibility:private"],
+    deps = [
+        "//contracts/deposit-contract:go_default_library",
+        "//proto/beacon/p2p/v1:go_default_library",
+        "//shared/keystore:go_default_library",
+        "//shared/ssz:go_default_library",
+        "//shared/version:go_default_library",
+        "@com_github_ethereum_go_ethereum//accounts/abi/bind:go_default_library",
+        "@com_github_ethereum_go_ethereum//accounts/keystore:go_default_library",
+        "@com_github_ethereum_go_ethereum//common:go_default_library",
+        "@com_github_ethereum_go_ethereum//crypto:go_default_library",
+        "@com_github_ethereum_go_ethereum//ethclient:go_default_library",
+        "@com_github_ethereum_go_ethereum//rpc:go_default_library",
+        "@com_github_sirupsen_logrus//:go_default_library",
+        "@com_github_urfave_cli//:go_default_library",
+        "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
+    ],
+)
+
+go_binary(
+    name = "sendDepositTx",
+    embed = [":go_default_library"],
+    visibility = ["//visibility:public"],
+)
+
+go_image(
+    name = "image",
+    srcs = ["sendDeposits.go"],
+    goarch = "amd64",
+    goos = "linux",
+    importpath = "github.com/prysmaticlabs/prysm/contracts/deposit-contract/sendDepositTx",
+    pure = "off",  # depends on cgo for go-etheruem crypto
+    race = "off",
+    static = "on",
+    tags = ["manual"],
+    visibility = ["//visibility:private"],
+    deps = [
+        "//contracts/deposit-contract:go_default_library",
+        "//shared/params:go_default_library",
+        "//shared/version:go_default_library",
+        "@com_github_ethereum_go_ethereum//accounts/abi/bind:go_default_library",
+        "@com_github_ethereum_go_ethereum//accounts/keystore:go_default_library",
+        "@com_github_ethereum_go_ethereum//crypto:go_default_library",
+        "@com_github_ethereum_go_ethereum//ethclient:go_default_library",
+        "@com_github_ethereum_go_ethereum//rpc:go_default_library",
+        "@com_github_sirupsen_logrus//:go_default_library",
+        "@com_github_urfave_cli//:go_default_library",
+        "@com_github_x_cray_logrus_prefixed_formatter//:go_default_library",
+        "@io_k8s_apimachinery//pkg/apis/meta/v1:go_default_library",
+        "@io_k8s_client_go//kubernetes:go_default_library",
+        "@io_k8s_client_go//rest:go_default_library",
```

Comment:
Why does the image target have a lot of deps that aren't present in the binary?

Reply:
This should be the same set of deps. Please ensure this manual target builds too.

```
bazel build //contracts/deposit-contract/sendDepositTx:image
```

Do we even need a docker image for this tool?

Reply:
ah, we do not need a docker image for this tool. It's supposed to be just for testing, and will not be used in production. For some reason `gazelle` didn't remove them

Reply:
Just realised I never removed the image in the build file

Reply:
Ok, please remove this target :)

Reply:
Done !